### PR TITLE
[IMP] l10n_ch: added English and updated translation

### PR DESCRIPTION
**Pull request by rgo-odoo**
- records in XML files are now in English language instead of French
- translation for the same will be loaded through *.po files for French, German and Italian
- improved translation for French, German and Italian languages

Task: 2391988

### CODE REVIEW

File: addons/l10n_ch/data/account.account.template.csv [hunk 1]
```diff
@@ -1,169 +1,169 @@
...
-"ch_coa_1099","Compte d'attente autre","1099","account.data_account_type_current_assets","l10n_ch.l10nch_chart_template","True"
-"ch_coa_1100","Débiteurs","1100","account.data_account_type_receivable","l10n_ch.l10nch_chart_template","True"
+"ch_coa_1060","Securities (with stock exchange price)","1060","account.data_account_type_current_assets","l10n_ch.l10nch_chart_template","False"
+"ch_coa_1069","Accumulated depreciation on securities","1069","account.data_account_type_current_assets","l10n_ch.l10nch_chart_template","False"
+"ch_coa_1091","Transfer account: Salaries","1091","account.data_account_type_current_assets","l10n_ch.l10nch_chart_template","True"
+"ch_coa_1099","Transfer account: miscellaneous","1099","account.data_account_type_current_assets","l10n_ch.l10nch_chart_template","True"
+"ch_coa_1100","Accounts receivable from goods and services (Debtors)","1100","account.data_account_type_receivable","l10n_ch.l10nch_chart_template","True"
 "ch_coa_1101","Débiteurs (PoS)","1101","account.data_account_type_receivable","l10n_ch.l10nch_chart_template","True"
```
Comment (jco-odoo) [inline addons/l10n_ch/data/account.account.template.csv:7]: Receivable (PoS)

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by rgo-odoo** [comment]
@jco-odoo  yes, it can be closed as it was taken care by #62152 and odoo/enterprise#16944

**Pull request closed without merging**